workspaces/Mynewt Nimble Host/Mynewt Nimble Host.pqw6

```diff
@@ -1924,7 +1924,7 @@
           <Row>
             <Name>TSPC_GATT_1a_2</Name>
             <Description>GATT Client over BR/EDR (C.2)</Description>
-            <Value>FALSE</Value>
+            <Value>TRUE</Value>

```

Comment:
Not sure how this change happen but we do not support BR/EDR, so this probably should be FALSE.